src/ott-analytics.js

```diff
@@ -91,6 +91,7 @@ class OttAnalytics extends BasePlugin {
    */
   destroy(): void {
     this._maybeSendStop();
+    this._clearMediaHitInterval();
```

Comment:
probably better call reset instead of what we do today, as it covers all(and more, which is not neccesery), but will also cover the code duplication issue.
Also, please add tests for both reset and destroy, so we don't miss this again 😬